EngineeringModel/ViewModels/OptionBrowser/OptionBrowserViewModel.cs

```diff
@@ -342,11 +342,14 @@ private void UpdateDefaultOption()
                 return;
             }
 
-            var row = this.Options.Single(x => x.Thing == defaultOption);
-            this.defaultOptionRow = row;
-            row.IsDefaultOption = true;
-        }
+            var row = this.Options.SingleOrDefault(x => x.Thing == defaultOption);
```

Comment:
There is a larger problem here. This is not the full solution. If you look at the code you see that if it got to this point it means that `defaultOption` is not null, which means that the iteration still has a `DefaultOption` property set, which, if you deleted the default option alltogether, should not be the case. So to me it looks like the server does not clean up properly when deleting an option. This fix is ok if you have broken data, but in my opinion theres a bug on the server somewhere.

Reply:
Yes, the bug on the server side is related to #107 which is already assigned to Kamil. I can take that over and fix it. I know where that is coming from as well.

Reply:
issue #107 has nothing to do with this. My guess is the number is wrong?

Reply:
Also a bit confused about your comment. The option can be set as default. For instance, we can add multiple options and one is set as default.

Reply:
https://github.com/RHEAGROUP/CDP4-WebServices-Community-Edition/issues/107

Reply:
After reviewing @kamilatrhea s fix, you are right they are connected.